.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.195.0/containers/go/.devcontainer/base.Dockerfile
+# [Choice] Go version (use -bullseye variants on local arm64/Apple Silicon): 1, 1.16, 1.17, 1-bullseye, 1.16-bullseye, 1.17-bullseye, 1-buster, 1.16-buster, 1.17-buster
+ARG VARIANT=1-bullseye
```

Comment:
Was this committed by accident? If not, this should probably be its own PR with a clear rationale why this needs to be in the `lnd` repo.

Reply:
Yep, that was totally a mistake. I love my devcontainers but sometimes they sneak into commits by mistake. Nice catch. My bad.